CHANGELOG.md

```diff
@@ -32,6 +32,9 @@ Full documentation for hipSOLVER is available at [hipsolver.readthedocs.io](http
 ### Deprecated
 ### Removed
 ### Fixed
+- bufferSize functions will now return HIPSOLVER_STATUS_NOT_INITIALIZED instead of HIPSOLVER_STATUS_INVALID_VALUE when both handle and lwork are null.
```

Comment:
Was the old behaviour somehow wrong? If so, a fix makes sense. I would have thought it was just a change.

Reply:
It's only wrong insofar as it doesn't follow rocSOLVER error precedence. In rocSOLVER, a null handle is always the first thing we check. I consider it a fix, but I guess it could go either way.